pixeltable/catalog/table_version_handle.py

```diff
@@ -31,6 +31,9 @@ def __eq__(self, other: object) -> bool:
             return False
         return self.id == other.id and self.effective_version == other.effective_version
 
+    def __hash__(self) -> int:
```

Comment:
Where is this being used?

Reply:
It's a standard programming practice to always implement `__hash__` any time `__eq__` is overridden. (Not just in Python, but in other languages such as Java too.)

This is to prevent the existence of objects that present as equal but have different hash codes, which can lead to _very_ strange bugs if they get used in collections. I suspect most linters in most languages will flag this.

I guess it's less of an issue in Python where non-hashable objects are a thing, but I also see no harm in following the linter and implementing it. (honestly I'm trained to automatically implement `__hash__` any time I implement `__eq__`, so I'm quite sympathetic to the linter here)